### PR TITLE
KubeVirt config provider

### PR DESCRIPTION
**Pull request by jakub-dzon**
This is a preparation for checking LiveMigration feature gate during placement policy validation.

```release-note
NONE
```

Signed-off-by: Jakub Dzon <jdzon@redhat.com>

### CODE REVIEW

File: pkg/controller/virtualmachineimport/virtualmachineimport_controller.go [hunk 1]
```diff
@@ -141,10 +151,15 @@ type ReconcileVirtualMachineImport struct {
 func (r *ReconcileVirtualMachineImport) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling VirtualMachineImport")
+	cfg, err := r.kvConfigProvider.GetConfig()
```
Comment (pkliczewski) [inline pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: Is there any reason to get it every time `Reconcile` is triggered?

File: pkg/controller/virtualmachineimport/virtualmachineimport_controller.go [hunk 2]
```diff
@@ -141,10 +151,15 @@ type ReconcileVirtualMachineImport struct {
...
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	reqLogger.Info("Using config", "KubeVirt", cfg)
```
Comment (pkliczewski) [inline pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: `cfg` is not used anywhere else than in the log.

Comment (jakub-dzon) [reply pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: It is here just to show what configuration is used. Placement policy validation implementation will come in another PR.

File: pkg/controller/virtualmachineimport/virtualmachineimport_controller.go (hunk 1 above)
Comment (jakub-dzon) [reply pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: It's there just for the time being - in the validation implementation PR it won't be there anymore.

File: pkg/controller/virtualmachineimport/virtualmachineimport_controller.go (hunk 2 above)
Comment (pkliczewski) [reply pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: As we know it works we can remove it in this PR and add it in another when we improve validation.

Comment (jakub-dzon) [reply pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: :+1:

Comment (jakub-dzon) [reply pkg/controller/virtualmachineimport/virtualmachineimport_controller.go]: Done

**Review by pkliczewski — APPROVED** [review]

**Review by machacekondra — APPROVED** [review]

**Pull request merged**